src/dataModels/SummarySubgrid.js

```diff
@@ -28,11 +28,11 @@ SummarySubgrid.prototype = {
     hasOwnData: true, // do not call setData implicitly
 
     getRowCount: function() {
-        return this.behavior.dataModel.getRowCount() && this.getData().length;
+        return this.getData().length;
     },
 
     getData: function() {
-        return this.behavior.dataModel.dataSource.getGrandTotals() || this.data;
+        return this.data;
```

Comment:
Despite the fact that this plugin won't be in core anymore, hence not of major concern, why are we reaching into one of a pipes for a method? The subgrids should not call non datasource-base interface methods

This is why I added these [line](https://github.com/openfin/fin-hypergrid/pull/474/files#diff-2a01e55e2f6e71aa32f816c37d8e76b7R107)